chain/rust/src/genesis/shelley/config.rs

```diff
@@ -18,7 +18,7 @@ pub struct ShelleyGenesisData {
     pub network_magic: u64,
     pub protocol_params: ShelleyGenesisProtocolParameters,
     pub security_param: u64,
-    pub slot_length: u64,
+    pub slot_length: String,
```

Comment:
Shouldn't this be a `Fraction` like `active_slots_coeff`? That one is only a `String` to avoid `f64` losing precision in `raw.rs`'s field, but is `Fraction` here.